test/v1/controllers/validators/RetrieveItsaStatusValidatorSpec.scala

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers.validators
+
+import api.models.domain.{Nino, TaxYear}
+import api.models.errors._
+import mocks.MockAppConfig
+import support.UnitSpec
+import v1.models.request.{RetrieveItsaStatusRawData, RetrieveItsaStatusRequest}
+
+class RetrieveItsaStatusValidatorSpec extends UnitSpec {
+
+  private val validNino    = "AA123456A"
+  private val validTaxYear = "2023-24"
+
+  private val invalidNino    = "Darth Sidious"
+  private val invalidTaxYear = "23-24"
+
+  private val validRawData: RetrieveItsaStatusRawData = RetrieveItsaStatusRawData(validNino, validTaxYear, None, None)
+
+  private val requestWithInvalidNino: RetrieveItsaStatusRawData =
```

Comment:
Because these are all private and the type is obvious, they don't really need the type declaration. Each one can then be on one line instead of two

Reply:
So it can all just be a single block like
```
  private val validRawData                     = RetrieveItsaStatusRawData(validNino, validTaxYear, None, None)
  private val requestWithInvalidNino           = RetrieveItsaStatusRawData(invalidNino, validTaxYear, None, None)
  private val requestWithInvalidTaxYear        = RetrieveItsaStatusRawData(validNino, invalidTaxYear, None, None)
  private val requestWithInvalidNinoAndTaxYear = RetrieveItsaStatusRawData(invalidNino, invalidTaxYear, None, None)
  private val requestWithInvalidFutureYears    = RetrieveItsaStatusRawData(validNino, validTaxYear, Some("yes"), None)
  private val requestWithInvalidHistory        = RetrieveItsaStatusRawData(validNino, validTaxYear, None, Some("yes"))
  private val requestWithAllInvalidRawData     = RetrieveItsaStatusRawData(invalidNino, invalidTaxYear, Some("yes"), Some("yes"))
```